### DISCUSSION TITLE
How can I have an image as footer in the bottom right corner?

### DISCUSSION THREAD

**Discussion by lucky-edu** [marp-team/marp#428] [Q&A] [answered]
Hello. First of all, ... amazing tool! I've been doing some great presentations in minutes for internal work.
I'm doing some classes and would love to use marp since all my material is already in markdown.
I need to include (see: <img width="1297" alt="image" src="https://user-images.githubusercontent.com/88039083/230652874-23ecd5d7-a898-443d-abfa-874e95e4cf69.png">) a footer image on the right bottom corner and I'm failing to do it.
I've checked the following documentation but no success:
- https://marp.app/
- https://marpit.marp.app/

Thanks in advance!

**Comment by yhatt** [comment, marked as answer]
Refer to #200. You may take one from several ways to put an image.

**Reply by lucky-edu** [reply]
thank you !! Exactly what I wanted. Have a great day :) !
